### PR TITLE
Updated yaml plugin to 0.13.0

### PR DESCRIPTION
**Pull request by tomzo**
Issue: #8266

Description:
Updates YAML plugin to 0.13.0 - which now supports format_version: 10.

What I tested:
Built GoCD from this branch.
Started a dev server.
Added a config repo with format_version:9 and some whitelist/blacklist uses in it
Renamed whitelist/blacklist to includes/ignore
Switched format version to 10

Commit history for the test: https://github.com/tomzo/gocd-yaml-config-example/commits/f10

Basically, a graceful way of upgrading and migrating the keywords will be possible now.

### CODE REVIEW

**Comment by marques-work** [comment]
@tomzo I think (if I understand the changes correctly) that MaterialTransform.java might have the wrong keyword (`excludes` vs `ignores`)?

See https://github.com/tomzo/gocd-yaml-config-plugin/pull/148#discussion_r488908236

**Comment by tomzo** [comment]
Thanks, let me investigate that

**Comment by tomzo** [comment]
@marques-work it turns out that this piece of code is irrelevant. I've started to cleanup the inverse transform code and I'll release another version later. But for now, this plugin version is functional and tested, so I would go with it for the 20.8 release.

Let me know if you're OK with that.

**Comment by marques-work** [comment]
@tomzo yes, I think that’s perfectly fine. I’ll merge this.

**Comment by maheshp** [comment]
Verified the bundled plugin version is `0.13.0` on `GoCD Version: 20.8.0 (12211-5d6b4a7a7145f728857afb8108b31effbbd1cd0c).`
